### ISSUE TITLE
Team 1: Implement get_avg_nums_of_comments function

### ISSUE THREAD

**Issue by mariakimheinert** [GatorGauge/GatorGauge#52]
`get_avg_nums_of_comments()` should take in a list of strings that are Java programs and return a dict that contains the average numbers of singleline, multiline, and Javadoc comments among all the strings. It should return something that looks like:
```
nums_of_comments = {"singleline": 4.52, "multiline": 3.93, "javadoc": 6.83}
```
The averages should be kept to 2 decimal places.

**Comment by mariakimheinert** [comment]
@colezAC how is this feature going?

**Comment by delgrecoj** [comment]
Addressed in PR #90.
